dora/core/server/master/src/test/java/alluxio/master/block/ConcurrentBlockMasterTest.java

```diff
@@ -732,8 +732,6 @@ public void concurrentRemoveWithRegisterNewWorkerDifferentBlock() throws Excepti
       long worker1 = registerEmptyWorker(NET_ADDRESS_1);
       // Prepare block on the worker
       mBlockMaster.commitBlock(worker1, BLOCK1_LENGTH, "MEM", "MEM", BLOCK1_ID, BLOCK1_LENGTH);
-      // Prepare block 2 so it is recognized at worker register
-      mBlockMaster.commitBlockInUFS(BLOCK2_ID, BLOCK2_LENGTH);
```

Comment:
this causes test to fail because the block does not exist
```
2023-07-18T22:39:36.5503269Z 22:39:36.548 [ERROR] alluxio.master.block.ConcurrentBlockMasterTest.concurrentRemoveWithRegisterNewWorkerDifferentBlock  Time elapsed: 1.436 s  <<< ERROR!
2023-07-18T22:39:36.5504152Z alluxio.exception.BlockInfoException: BlockMeta not found for blockId 2
2023-07-18T22:39:36.5504761Z 	at alluxio.master.block.DefaultBlockMaster.lambda$getBlockInfo$7(DefaultBlockMaster.java:1178)
2023-07-18T22:39:36.5505283Z 	at java.util.Optional.orElseThrow(Optional.java:290)
2023-07-18T22:39:36.5505844Z 	at alluxio.master.block.DefaultBlockMaster.getBlockInfo(DefaultBlockMaster.java:1178)
2023-07-18T22:39:36.5506545Z 	at alluxio.master.block.BlockMasterTestUtils.verifyBlockOnWorkers(BlockMasterTestUtils.java:30)
2023-07-18T22:39:36.5507462Z 	at alluxio.master.block.ConcurrentBlockMasterTest.lambda$concurrentRemoveWithRegisterNewWorkerDifferentBlock$29(ConcurrentBlockMasterTest.java:783)
2023-07-18T22:39:36.5508420Z 	at alluxio.master.block.ConcurrentBlockMasterTest.concurrentWriterWithWriter(ConcurrentBlockMasterTest.java:1145)
2023-07-18T22:39:36.5509489Z 	at alluxio.master.block.ConcurrentBlockMasterTest.concurrentRemoveWithRegisterNewWorkerDifferentBlock(ConcurrentBlockMasterTest.java:740)
2023-07-18T22:39:36.5510311Z 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2023-07-18T22:39:36.5510883Z 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2023-07-18T22:39:36.5511528Z 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2023-07-18T22:39:36.5512061Z 	at java.lang.reflect.Method.invoke(Method.java:498)
2023-07-18T22:39:36.5512567Z 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
2023-07-18T22:39:36.5513190Z 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
2023-07-18T22:39:36.5513856Z 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
2023-07-18T22:39:36.5514518Z 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
2023-07-18T22:39:36.5515150Z 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
2023-07-18T22:39:36.5515778Z 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
2023-07-18T22:39:36.5516344Z 	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
2023-07-18T22:39:36.5516940Z 	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:258)
2023-07-18T22:39:36.5517739Z 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
2023-07-18T22:39:36.5518301Z 	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
2023-07-18T22:39:36.5518842Z 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
2023-07-18T22:39:36.5519502Z 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
2023-07-18T22:39:36.5520296Z 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
2023-07-18T22:39:36.5520833Z 	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
2023-07-18T22:39:36.5521392Z 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
2023-07-18T22:39:36.5521986Z 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
2023-07-18T22:39:36.5522502Z 	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
2023-07-18T22:39:36.5522988Z 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
2023-07-18T22:39:36.5523556Z 	at alluxio.heartbeat.ManuallyScheduleHeartbeat$1.evaluate(ManuallyScheduleHeartbeat.java:81)
2023-07-18T22:39:36.5524082Z 	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
2023-07-18T22:39:36.5524541Z 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
2023-07-18T22:39:36.5525017Z 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
2023-07-18T22:39:36.5525564Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:316)
2023-07-18T22:39:36.5526246Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:240)
2023-07-18T22:39:36.5527195Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:214)
2023-07-18T22:39:36.5527852Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:155)
2023-07-18T22:39:36.5528528Z 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
2023-07-18T22:39:36.5529199Z 	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
2023-07-18T22:39:36.5529801Z 	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
2023-07-18T22:39:36.5530371Z 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
```

what about keeping `commitBlockInUFS()` as a private function in `DefaultBlockMaster` just for test preparation? I feel we should do the minimal in surgery and do not add new code in this kind of PRs.

Reply:
fixed

Reply:
forgot to git push?